app/assets/javascripts/discourse/lib/render-tags.js.es6

```diff
@@ -43,7 +44,7 @@ export default function(topic, params){
     buffer = "<div class='discourse-tags'>";
     if (tags) {
       for(let i=0; i<tags.length; i++){
-        buffer += renderTag(tags[i]) + ' ';
+        buffer += renderTag(tags[i], { isPrivateMessage: isPrivateMessage }) + ' ';
```

Comment:
I think `{ isPrivateMessage: isPrivateMessage }` can be written simply as `{ isPrivateMessage }`. ([Reference documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer).)